docs/kubernetes.md

```diff
@@ -0,0 +1,101 @@
+# Running on Kubernetes
+
+Daft can run on Kubernetes using the Daft quickstart Helm chart. This allows you to test drive both simple single-node jobs and distributed Ray-based workloads on your Kubernetes cluster without any additional dependencies.
+
+!!! warning "Not for Production"
+
+    The Daft quickstart is designed for development and experimentation. For running production Daft workloads on Ray clusters on Kubernetes, use [KubeRay](https://ray-project.github.io/kuberay/), the official Ray operator for Kubernetes.
+
+## Prerequisites
+
+- Kubernetes 1.19+
+- Helm 3.x
+- kubectl configured to access your cluster
+
+## Simple Mode
+
+Run a quick Daft job on Kubernetes using the native runner:
+
+=== "🐍 Python"
+
+    ```python
+    # Create a script file: my_script.py
+    # /// script
+    # dependencies = ["daft"]
+    # ///
+    import daft
+
+    # Set IO config for anonymous S3 access
+    daft.set_planning_config(
+        default_io_config=daft.io.IOConfig(s3=daft.io.S3Config(anonymous=True))
+    )
+
+    df = daft.read_parquet(
+        "s3://daft-public-data/tutorials/10-min/sample-data-dog-owners-partitioned.pq/**"
+    )
+
+    df.where(daft.col("age") >= 40).show()
```

Comment:
Can we fix our examples to be the same? No reason to have them be different across simple and distributed mode